### PR TITLE
344/bruteforce/organizer on event page

### PR DESCRIPTION
**Pull request by laurenzse**

### CODE REVIEW

File: features/events/show.feature [hunk 1]
```diff
@@ -18,3 +21,11 @@ Feature: Detailed event page
     Given a league without max teams
     When the league's page is visited
     Then the page should show 'unbegrenzt'
+
+  Scenario: Organizer name displayed
+    Given an Event e
+    And user o is organizer of this event
+    When visiting the event page of e
+    Then o should be listed as organizer
+    And clicking on the name should lead to the profile page of o
```
Comment (Askir) [inline features/events/show.feature]: Implement the steps for this please :)

**Review by Askir — CHANGES_REQUESTED** [review]
Implement the cucumber steps for this please :) Otherwise looks quite good!

**Comment by laurenzse** [comment]
Sorry for the combined merge and cucumber commit. Didn't commit after merging, then my latest commit got messed up.

File: features/step_definitions/events.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  event = event_named eventName
+  create_user_named userName
+  user = user_named userName
+  event.editors << user
+  user
+end
+
+Then(/^(.*) should be listed as organizer$/) do |userName|
+  user = user_named userName
+  expect(page).to have_text(user.name)
```
Comment (laurenzse) [inline features/step_definitions/events.rb:15]: Does anyone have a better idea on how to specifically look for organizers? They are displayed as a list in a `description list`, so a simple `have_text: Organizer:...` won't work.

Comment (Askir) [reply features/step_definitions/events.rb:15]: fine for now imo.

**Review by Askir — APPROVED** [review]

**Pull request merged**